test/unit/BCoWPool.t.sol

```diff
@@ -141,6 +157,142 @@ contract BCoWPool_Unit_EnableTrading is BaseCoWPoolTest {
   }
 }
 
+contract BCoWPool_Unit_Verify is BaseCoWPoolTest, SwapExactAmountInUtils {
+  function setUp() public virtual override(BaseCoWPoolTest, BasePoolTest) {
+    BaseCoWPoolTest.setUp();
+  }
+
+  function _assumeHappyPath(SwapExactAmountIn_FuzzScenario memory _fuzz) internal view override {
```

Comment:
would you have preferred to template-method the implementation for `BPool` and only define the different parts here? I tought the generalization on `BPool` would be pretty inelegant and coupled to what I want to do here.

Reply:
...or I could've just copy-pasted the implementation into the testcase instead

Reply:
i think is wise to reuse the `_setValues`, tho the happy path shouldn't be a blind copy of the methods, there's some details on #104